ocpi-toolkit-2.2.1/src/test/kotlin/com/izivia/ocpi/toolkit/samples/common/Http4kTransportServer.kt

```diff
@@ -63,7 +60,9 @@ class Http4kTransportServer(
                         }
                         .also { httpRequest -> filters.forEach { filter -> filter(httpRequest) } }
                         .let { httpRequest ->
-                            httpRequest to runBlocking {
+                            httpRequest to runBlocking(
+                                httpRequest.messageRoutingHeaders() + ResponseMessageRoutingHeaders()
```

Comment:
should we "invert" by default. unless you implement a HUB, a server would always need to mirror the clients routing headers. Similar to how we already handle request/response  message ID headers automatically.

Reply:
Absolutely, good suggestion!